index.rst

```diff
@@ -60,9 +68,8 @@ that welcomes community projects, contributions, suggestions, fixes and
 constructive feedback.
 
 If you find any errors or have suggestions for improvements, please use the
-"Give feedback" button at the top of every page. This will take you to
-GitHub where you can share your comments or let us know about bugs with any
-page.
+links at the bottom of every page. There you can ask questions, file issues,
```

Comment:
I'm happy for us to add a note pointing out the links at the bottom of the page, but I'd prefer it to be in addition to (not instead of) the request to use the "give feedback" button, since this is linked to the issue template. The issues we get through the feedback button give us better/more information.

Reply:
I can believe it, I do that all the time :)